recipes/required-and-default-props.md

```diff
@@ -31,7 +31,7 @@ import 'package:flutter/material.dart';
 class SomeComponent extends StatelessWidget {
   SomeComponent({
     @required this.foo,
-    this.bar = 'some string',
+    this.bar : 'some string',
```

Comment:
hm, both work fine. Do you know the difference between `=` vs `:`? just curious

Reply:
Yeah, u right: official documentation say about u can use '=', but i'm not sure is using it in this case is the best practice.